class/Zarieth; The Witch

```diff
@@ -0,0 +1,1836 @@
+										[
+											"3rd",
+											"{@spell Blur},{@spell Calm Emotions}"
+										],
+										[
+											"5th",
+											"{@spell Feign Death}, {@spell Spirit Guardians}"
+										],
+										[
+											"7th",
+											"{@spell Death Ward}, {@spell Freedom of Movement}"
+										],
+										[
+											"9th",
+											"{@spell Commune}, {@spell Legend Lore}"
+										]
+									]
+								},
+								{
+									"name": "Spirit Link",
+									"entries": 
+									[
+										"You can link yourself to the souls of others, gaining knowledge of them. Beginning at 2nd level, while in your quickening you can use your action to learn all conditions currently afflicting it, as well as the school of magic of any spell or spells currently affecting it."
+									]
+								}
+							]
+						}
+					],
+					[
+						{
+							"entries": [
+								{
+									"name": "Forceful Presence",
+									"entries": [
+										"When you reach 6th level, you can retain control over a spirit after it produces its release effect so that it remains bound and ready to be released again.",
+										"You may use this ability a number of times equal to your Charisma modifier (minimum of once). You recover all expended uses after a long rest."
+									]
+								}
+							]
+						}
+					],
+					[
+						{
+							"entries": 
+							[
+								{
+									"name": "Powerful Bonds",
+									"entries": 
+									[
+										"At 10th level, your body and soul develop the ability to contain more powerful spirits. The level of spirits you bind are considered 1 higher than the spell slot you use to bind them."
+									]
+								}
+							]
+						}
+					],
+					[
+						{
+							"entries": 
+							[
+								{
+								"name": "Pale Host",
+								"entries": 
+									[
+										"At 14th level, when you are reduced to 0 hit points but not killed outright, your body becomes possessed by a spirit you control and you may continue to act for 1 round per level of the highest level spirit you have in your power. You cannot use this ability if you do not have any spirits bound. During this time, your hit points remain at 0 and you continue to make death saving throws (though a prepared witch may move to her spirit ward to become stable).",
+										"Onlookers notice a strangeness in your movements, eyes, and voice while you remain possessed. While possessed, you can use your action on your turn to activate the release power of one of the spirits you have bound and have it remain bound. This does not expend any uses of the Forceful Presence ability.",
+										"Should you be healed under the possession (including by yourself), you awaken from the possession, conscious. Otherwise, when the duration expires, your body collapses."
+									]
+								}
+							]
+						}
+					]
+				]
+			}
+		]
+	}
+
+],
+"optionalfeature": 
+[
+	{
+		"type": "optfeature",
+		"name": "Agony",
+		"source": "The Witch",
```

Comment:
This should match your `"json"` source defined in `"_meta"`, e.g. be `"ZariethTheWitchFromGMBinder"` (and the same for the others)

Reply:
Properly renamed them in all instances.